src/components/AddBook.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function AddBook() {
+  return (
+    <div>
+      <h2>ADD NEW BOOK</h2>
+      <div>
+        <input type="text" placeholder="Book title" />
+        <input type="text" multiple list="drawCategories" placeholder="Category" />
+        <datalist id="drawCategories">
+          <option>Action</option>
+        </datalist>
+        <button type="button">ADD BOOK</button>
+      </div>
```

Comment:
- [ ] Please let's stick to the requirements of this project which says this should be a `form` (not a `div` as in your case) with inputs(You have thems already) for the title :+1: and author(missing) of the book. Let's also use `submit` as the value of the button type ie `<button type = "submit".....

Reply:
Thank you so much for your feedback @FrederickMih! I really appreciate your comments looking to improve my project.
😄